webknossos/webknossos/dataset/dataset.py

```diff
@@ -2794,7 +2794,9 @@ def _export_as_json(self) -> None:
                 json.dump({"zarr_format": 3, "node_type": "group"}, outfile, indent=4)
 
         for layer in self.layers.values():
-            write_ome_0_4_metadata(self, layer)
+            # Only write out OME metadata if the layer is a child of the dataset
+            if layer.path.resolve().parent == self.path:
```

Comment:
Could `self.path` also be a symlink in which case it should also be resolved for this comparison?

Reply:
Good point. I also added caching for the path resolution, because on remote systems that may be one request per layer on every save.